Changelog.md

```diff
@@ -16,6 +16,7 @@
 - 60FPS: Fix model position glitch for some summons
 - 60FPS: Fix FIELD 3D model blinking time and wait time
 - FF7 Achievement: Fix bug about Cait Sith last limit break and Emerald Weapon achievements not achievable
+- FF7 Achievement: Fix crash during load save file for spanish version
```

Comment:
I think also this line is not required, as we have the very same one just two lines below

Reply:
Ops, didn't saw that line

Reply:
Thank you